podcasts/Onboarding/Import/ImportViewModel.swift

```diff
@@ -34,10 +34,11 @@ class ImportViewModel: OnboardingModel {
         .init(id: .castbox, displayName: "Castbox", steps: L10n.importInstructionsCastbox),
         .init(id: .overcast, displayName: "Overcast", steps: L10n.importInstructionsOvercast),
         .init(id: .other, displayName: "other apps", steps: L10n.importPodcastsDescription),
+        .init(id: .opmlFromURL, displayName: "URL", steps: L10n.importOpmlFromUrl)
     ]
 
     enum ImportAppId: String, AnalyticsDescribable {
```

Comment:
Now this is not only related to apps, maybe renaming it to `ImportSourceId` makes more sense? Wdyt?

Reply:
That lead me to replace all references to app to import source. Instead of installedApps, Im looking for AvailableResources